crates/hc_sandbox/tests/cli.rs

```diff
@@ -71,10 +70,32 @@ async fn get_app_info(admin_port: u16, installed_app_id: InstalledAppId, port: u
         .await
         .unwrap();
 
-    let request = AppRequest::AppInfo;
-    let response = app_tx.request(request);
-    let r: AppResponse = check_timeout(response).await;
-    assert_matches!(r, AppResponse::AppInfo(Some(_)));
+    tokio::time::timeout(Duration::from_secs(60), async move {
```

Comment:
@ThetaSinner here. It already contains the test fix here with the small difference that a value is being returned.